dataland-frontend/src/components/forms/CreateLksgDataset.vue

```diff
@@ -0,0 +1,903 @@
+<template>
+  <Card class="col-12 page-wrapper-card">
+    <template #title>New Dataset - LkSG </template>
+    <template #content>
```

Comment:
Company Legal Form is part of the company upload, right? Therefore it is not part of LKSG upload form?

Reply:
Yes it is part of the company upload. But I don't understand the context of your question.
Why should it be part of the LkSG form?

Reply:
ok